documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -14,290 +14,210 @@ keywords:
+Once the transaction is confirmed, the DID is published and can be formatted by using the `Alias ID` as the tag in [DID Format](#did-format).
 
 ### Read
 
-To read the latest DID Document associated with a DID, the following steps are to be performed:
-1. Query all the Integration DID messages from the index, which is the `tag` part of the DID.
-2. Order the messages based on `previousMessageId` linkages. See [Determining Order](#determining-order) for more details.
-3. Validate the first Integration DID message containing a verification method with a public key that, when hashed using the `BLAKE2b-256` hashing algorithm, equals the `tag` field of the DID as per [generation](#generation). This first Integration DID message MUST also contain a valid signature referencing the same verification method.
-4. Verify the signatures of all the DID messages. Signatures must be created using a public key available in the previous DID message linked to a verification method capable of signing DID messages. See [Signing Keys](#signing-keys).
-5. Ignore any messages that are not signed correctly, afterwards ignore any messages that are not first in the ordering for their specific location in the chain.
-6. Query all the Diff DID messages from the index, generated by the MessageId from the last valid Integration DID message, hashed using `Blake2b-256` and encoded in `hex`.
-7. Order messages and validate signatures in a similar manner to steps 2, 4 and 5 above. The first valid Diff DID message MUST have a `previousMessageId` referencing the `messageId` of the last Integration DID message. 
-8. Ignore Diff DID messages with illegal operations such as removing or updating a signing key.
-9. Apply all valid Diff updates to the state generated from the Integration DID messages. This will result in the latest DID Document.
-
-#### Determining Order
-
-To determine the order of any DID messages, the following steps are to be performed:
-1. Order is initially established by recreating the chain based on the `previousMessageId` linkages. 
-   1. For Int DID messages, the one with no `previousMessageId` is first.
-   2. For Diff DID messages, the one with a `previousMessageId` referencing the IOTA MessageId of the corresponding Integration DID message is first.
-2. When two or more Messages compete, an order must be established between them.
-3. To determine the order, check which milestone confirmed the messages.
-4. If multiple messages are confirmed by the same milestone, we order based on the IOTA MessageId with alphabetical ordering.
+The following steps can be used to read the latest DID document associated with a DID.
+1. Obtain the `Alias ID` from the DID by extracting the `iota-tag` from the DID, see [DID Format](#did-format).
+2. Query the Alias Output corresponding to the `Alias ID`, using a node running the [inx indexer](https://github.com/iotaledger/inx-indexer). Nodes usually include this indexer by default.
```

Comment:
Step 5 might seem odd, but since I would suggest to no longer derive the final DID from the alias id of the output, as stated elsewhere, we should at least do this basic check.

```suggestion
1. Obtain the `Alias ID` from the DID by extracting the `iota-tag` from the DID, see [DID Format](#did-format).
2. Obtain the network of the DID by extracting the `iota-network` from the DID, see [DID Format](#did-format).
3. Query the Alias Output corresponding to the `Alias ID` using a node running the [inx indexer](https://github.com/iotaledger/inx-indexer). Nodes usually include this indexer by default.
4. Assert that the extracted network matches the one returned from the node. Return an error otherwise.
5. Assert that the `Alias ID` of the returned output matches the `Alias ID` extracted from the DID. Return an error otherwise.
```

Reply:
> Query the Alias Output corresponding to the `Alias ID` using a node running the [inx indexer]

> Assert that the extracted network matches the one returned from the node. Return an error otherwise.

We should probably state which REST APIs to call?